envVariables.js

```diff
@@ -39,6 +39,10 @@ let enviromentVariables = {
   "USER_SERVICE_URL" : {
     "message" : "Required user service base url",
     "optional" : false
+  },
+  "SERVICE_NAME" : {
```

Comment:
@VISHNUDAS-tunerlabs  add default value hence service name will be same in all env